mapproxy/service/tile.py

```diff
@@ -204,7 +204,7 @@ def _render_root_resource_template(self, service):
 
 
 class TileLayer(object):
-    def __init__(self, name, title, md, tile_manager, info_sources=[], dimensions=None):
+    def __init__(self, name, title, md, tile_manager, info_sources=[], dimensions=None, legend_version=None):
```

Comment:
Not directly related but it is not recommended to use `[]` as default values since the default `[]` will be a single instance for all calls. This might lead to call by reference errors. Something like this is more stable:

```
def __init__(self, name, title, md, tile_manager, info_sources=None, dimensions=None, legend_version=None):
  if info_sources is None:
    info_sources = []
```

Reply:
good information. this should be addressed in the whole code base I think. Maybe we do an issue for this one and solve it separately.